modules/ROOT/partials/misc/mobile-platform-compatibility.adoc

```diff
@@ -1,12 +1,39 @@
-== Supported Mobile Platforms
+== Supported Mobile Platforms (1)
```

Comment:
Remove ` (1)`

Reply:
Once one section gets all the appropriate confirmations and blessings, the identifying sub-string in the section that goes into production will have the identifier removed.